src/main/scala/cognite/spark/v1/FlexibleDataModelRelationUtils.scala

```diff
@@ -93,7 +84,9 @@ object FlexibleDataModelRelationUtils {
           externalId = externalId,
           instanceSpace = space,
           source,
-          typeDirectRelation =
+          nodeTypeDirectRelation =
+            extractNodeTypeDirectRelation(schema, instanceSpace.orElse(Some(space)), row).toOption,
+          edgeTypeDirectRelation =
```

Comment:
This is the only functional change. By doing this, I make sure that if writing a node we are only getting from "_type" and not from "type" even on a nodeOrEdge view.

Before if you had a nodeOrEdge view and were writing to a node instance implementing this view, and the view had a "type" attribute you would encounter an issue, this should fix that problem.

The reason this fix was not higher priority is because I don't think it actually happens for anyone -yet- NodeOrEdges are rare in fdm and attributes of views named "type" are too.